package.json

```diff
@@ -2,8 +2,11 @@
   "name": "root",
   "private": true,
   "license": "Apache-2.0",
+  "workspaces": [
+    "packages/*",
+    "examples/*"
+  ],
   "scripts": {
-    "postinstall": "lerna bootstrap --no-ci",
```

Comment:
removed postInstall for lerna bootstrap as no longer needed with lerna workspaces.